src/pages/ProfilePage/views/MyHousesView/MyHousesView.tsx

```diff
@@ -1,37 +1,28 @@
-import MyHousesListItem from '../../../../components/MyHousesList/ListItem/MyHousesListItem';
-import MyHousesList from '../../../../components/MyHousesList/MyHousesList';
+import MyHousesListItem from 'src/components/ProfilePage/MyHousesList/ListItem/MyHousesListItem';
+import MyHousesList from 'src/components/ProfilePage/MyHousesList/MyHousesList';
```

Comment:
There are still some warnings about the order of import. Running `prettier` would fix this automatically.